### ISSUE TITLE
CMD-Z issue - please bounce to line that changed

### ISSUE THREAD

**Issue by sascha1337** [decosoftware/deco-ide#130]
if i use cmd-z, i am used to see what line changed. when i had changes on the top of the file but viewing the bottom and i run cmd-z i have to guess because its not bouncing to the changed line.
please fix this because i run always in this issue

**Comment by dabbott** [comment]
Yep, definitely an issue. It happens because we only respect codemirror changes that affect the actual text content at the moment. Need to also respect editor selection.

IMO not too hard to fix. I'll try to get to it when I take a pass over editor stuff. Someone else is welcome to give it a shot - but it's in the depths of some of our more complex code, so I think it'd be hard to jump into.

**Comment by sascha1337** [comment]
thank you very much 🌴
